### PR TITLE
RTL: use backslash to write fractions (grades)

### PR DESCRIPTION
**Pull request by ARMBouhali**
**TL;DR -** This commit replaces the slash `/` for RTL languages with a backslash `\` when it is used to write fractions for such as `2/5` (grades)

**Background**
- In fractions, the direction of the slash determines which number comes on top (numerator) and which is on the bottom (denominator). For example: 3 / 2 means "2 over 3,  "2 divided by 3" or "2 out of 3"
- In RTL text, some characters are mirrored to match the writing direction, such as parentheses `( )` and comparison signs `< >`
- However, `/` is not mirrored in RTL, which introduces visual confusion when used to write fractions. For Example, **2 / 3** in English still shows as **2 / 3** in Arabic, while it should look **3 \ 2**

**What changed?**
- We use 2 frontend-platform functions `isRtl` and `getLocale` to apply the fix only if the current user language is right-to-left.
- we replace `/` with `\` for RTL languages in places where absolute grades are shown.
- The change is applied in the gradebook table when the score view is set to **absolute**, and in the **Edit grades modal > Adjusted grade column**

**Screenshots**
1. **Gradebook table with absolute score view**

| LTR for reference | RTL before fix | RTL after fix |
| ---- | ---- | ---- |
| Screenshot from 2022-09-29 11-42-09 | Screenshot from 2022-09-29 11-44-11 | Screenshot from 2022-09-29 11-42-48 |

2. **Adjusted grade input**
Look at the slash under the grade input field

| LTR for reference | Screenshot from 2022-09-29 11-47-16 |
| --- | --- |
| RTL before fix | Screenshot from 2022-09-29 11-45-52 |
| RTL after fix | Screenshot from 2022-09-29 11-46-18 |

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [x] Received code-owner approving review

**Testing Instructions**
1. Change the platform language to an RTL language (eg Arabic)
5. Choose an existing course with at least one enrolled student, then access its gradebook app page.
6. Set the score view to **absolute**, then In the gradebook table check the grades are rendered with backslashes `\` instead of slashes `\`
7. In the **Edit grades modal > Adjusted grade column**, the possible grade should be preceded by a backslash.

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ARMBouhali!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

**Comment by natabene** [comment]
@ARMBouhali Thanks for the contribution. We are waiting for the agreement to be sorted out.

**Comment by natabene** [comment]
@ARMBouhali Great, your CLA check is green. This can be reviewed by our team.

**Comment by schenedx** [comment]
@ARMBouhali Could you please update your commit message to follow conventional commits? That make make the merge check green. 
Also, please attach a screenshot of the UI after your change.

**Comment by ARMBouhali** [comment]
@schenedx I've fixed the commit message and updated the PR description with screenshots.

**Review by schenedx — APPROVED** [review]
LGTM 👍

**Comment by openedx-webhooks** [comment]
@ARMBouhali 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-gradebook&entry.867055334=https://github.com/openedx/frontend-app-gradebook/pull/266&entry.1484655318=https://github.com/ARMBouhali&entry.752974735=2022-09-15+14:18&entry.1917517419=2022-09-29+19:02&entry.2133058324=Yes) so we can improve your experience in the future.
